Library/Homebrew/dev-cmd/pr-automerge.rb

```diff
@@ -33,12 +35,13 @@ def pr_automerge
     pr_automerge_args.parse
 
     ENV["HOMEBREW_FORCE_HOMEBREW_ON_LINUX"] = "1" unless OS.mac?
-    with_label = Homebrew.args.with_label || "ready to merge"
     without_labels = Homebrew.args.without_labels || ["do not merge", "new formula"]
     tap = Tap.fetch(Homebrew.args.tap || CoreTap.instance.name)
 
-    query = "is:pr is:open repo:#{tap.full_name} label:\"#{with_label}\""
-    query += args.ignore_failures? ? " -status:pending" : " status:success"
+    query = "is:pr is:open repo:#{tap.full_name}"
+    query += Homebrew.args.ignore_failures? ? " -status:pending" : " status:success"
+    query += " review:approved" if Homebrew.args.only_approved?
+    query += " label:\"#{with_label}\"" if Homebrew.args.with_label
```

Comment:
Does that mean the default behaviour is to merge everything open with passing tests? Sounds dangerous, even if it may never be invoked manually

Reply:
`pr-automerge` does not merge anything if one does not pass `--publish` option.

Reply:
We can make the `review:approved` query default and just remove the option altogether, if you think it's better.

Reply:
Yes, that'd be great 👍